upstairs/src/lib.rs

```diff
@@ -6753,6 +6826,24 @@ pub async fn up_main(opt: CrucibleOpts, guest: Arc<Guest>) -> Result<()> {
         up_ds_listen(&upc, ds_done_rx).await;
     });
 
+    /*
+     * spawn a task to register with Oximeter and handle it.
+     */
+    let up_oxc = Arc::clone(&up);
+    let ups = up_oxc.stats.clone();
+    tokio::spawn(async move {
+        // XXX How do I find the IP of propolis or whomever I'm
+        // running on?
+        let my_addr: SocketAddr = "127.0.0.1:55443".parse().unwrap();
+        // XXX How do we get the correct address for registering with nexus?
+        let reg_addr: SocketAddr = "127.0.0.1:12221".parse().unwrap();
```

Comment:
Not real sure if this will work with a real nexus setup.  It does work when I run the simulated sled agent and associated parts on a helios system, but I suspect the IPv4 addresses won't work.